### ISSUE TITLE
grpc.load Deprecation warning displayed

### ISSUE THREAD

**Issue by DominicKramer** [googleapis/nodejs-bigtable#301]
When using this library, the following warning is printed to standard out:
```
(node:230621) DeprecationWarning: grpc.load: Use the @grpc/proto-loader module with grpc.loadPackageDefinition instead
```

**Comment by JustinBeckwith** [comment]
This is caused by gax-nodejs.  @alexander-fenster is actively working on a fix :)

**Comment by raulvillora** [comment]
I am getting that warning, but I'm not sure if that warning is causing me the error display after:

Server is listening on port: 1337
(node:22126) DeprecationWarning: grpc.load: Use the @grpc/proto-loader module with grpc.loadPackageDefinition instead
events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE :::1337
    at Object._errnoException (util.js:992:11)
    at _exceptionWithHostPort (util.js:1014:20)
    at Server.setupListenHandle [as _listen2] (net.js:1355:14)
    at listenInCluster (net.js:1396:12)
    at Server.listen (net.js:1480:7)
    at Object.<anonymous> (/Users/raulvilloravalencia/OneDrive - Universidad Politécnica de Madrid/Documents/cp-test-app-2/cp-test-app/server.js:171:8)
    at Module._compile (module.js:652:30)
    at loader (/Users/raulvilloravalencia/OneDrive - Universidad Politécnica de Madrid/Documents/cp-test-app-2/cp-test-app/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/raulvilloravalencia/OneDrive - Universidad Politécnica de Madrid/Documents/cp-test-app-2/cp-test-app/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:565:32)

**Comment by raulvillora** [comment]
So I am not sure if the warning is causing me the error or they are different things, I mean one is a display warning and the other is an error not related to that warning.

**Comment by raulvillora** [comment]
Now works. It was showing the port was already in used. I change the port and start the server again.

**Comment by JustinBeckwith** [comment]
This was fixed with the 1.0 release 😄

**Comment by calibri87** [comment]
@raulvillora when you say port numnber - how and where did you change this? was this in your functions package.json file?
